sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -180,29 +211,32 @@ def main(config:str,
                 config_dict["skip_bad_books"] = False
             #Adding info that should be there for all observations
             #Descriptive string columns
-            frequent_cols = ["telescope", 
-                             "telescope_flavor", 
-                             "tube_slot", 
-                             "tube_flavor", 
-                             "detector_flavor"]
-            for fc in frequent_cols:
-                fcvalue = index.get(fc)
-                if fcvalue is not None:
-                    bookcartobsdb.add_obs_columns([fc+" str"])
-                    very_clean[fc] = fcvalue
+            try:
+                telescope = index["telescope"]
+                flavors = telescope_lookup(telescope)
+                for flav in flavors:
+                    bookcartobsdb.add_obs_columns([flav+" str"])
+                    very_clean[flav] = flavors[flav]
+
+            except KeyError:
+                logger.warning("No telescope key in index file")
+                pass
```

Comment:
So we're not going to make this a fatal error?  I guess that's ok, but maybe in the failed case, at least fill in very_clean['telescope_flavor'] = 'unknown', so that we can sort of see what happened in the output obsdb.

In an except block, you don't need "pass" if there are other statements.